experiments/plot_training.ipynb

```diff
@@ -60,9 +51,9 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.7.7"
   }
  },
  "nbformat": 4,
- "nbformat_minor": 1
+ "nbformat_minor": 4
 }
```

Comment:
Need to remove these changes from this PR. Can clean up this notebook when I remove Jax.